bitcoin/src/taproot/merkle_branch/buf.rs

```diff
@@ -126,7 +126,7 @@ impl_try_from!(&[TapNodeHash]);
 impl_try_from!(Vec<TapNodeHash>);
 impl_try_from!(Box<[TapNodeHash]>);
 
-macro_rules! impl_try_from_array {
+macro_rules! impl_from_array {
```

Comment:
Thanks, though I planned to join it with the other macro. Not a big deal, I'll do it later.